src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/TraceEventExtensions.cs

```diff
@@ -22,6 +22,11 @@ public static bool TryGetCounterPayload(this TraceEvent traceEvent, CounterFilte
                 //Make sure we are part of the requested series. If multiple clients request metrics, all of them get the metrics.
                 string series = payloadFields["Series"].ToString();
                 string counterName = payloadFields["Name"].ToString();
+
+                //CONSIDER
+                //Concurrent counter sessions do not each get a separate interval. Instead the payload
+                //for _all_ the counters changes the Series to be the lowest specified interval.
+                //This means concurrent counter sessions may have missing data.
```

Comment:
Potentially worth clarifying this is scoped to a provider and that different providers can be configured with different intervals. Also worth mentioning that the missing data would be due to filtering in these pipelines rather than the data streams.